Index: trunk/webroot/query.php
===================================================================
--- trunk/webroot/query.php	(revision 64)
+++ trunk/webroot/query.php	(working copy)
@@ -22,284 +22,344 @@
 $xmlrpc_server = xmlrpc_server_create();
 
 xmlrpc_server_register_method($xmlrpc_server, "dir_places_query",
-		"dir_places_query");
+                "dir_places_query");
 
 function dir_places_query($method_name, $params, $app_data)
 {
-	$req 			= $params[0];
+ $req            = $params[0];
 
-	$text 			= $req['text'];
-	$category 		= $req['category'];
-	$query_start 	= $req['query_start'];
+ $text           = $req['text'];
+ $category       = $req['category'];
+ $query_start    = $req['query_start'];
 
-	if ($text == "%%%")
-	{
-		$response_xml = xmlrpc_encode(array(
-				'success'	  => False,
-				'errorMessage' => "Invalid search terms"
-		));
+ if ($text == "%%%")
+ {
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => False,
+                 'errorMessage' => "Invalid search terms"
+                 ));
 
-		print $response_xml;
+ print $response_xml;
 
-		return;
-	}
+ return;
+}
 
-	if ($category != -1)
-	{
-		$result = mysql_query("select * from parcels where " .
-				"(searchcategory = -1 or searchcategory = '" .
-				mysql_escape_string($category) ."') and (parcelname like '%" .
-				mysql_escape_string($text) . "%' or description like '" .
-				mysql_escape_string($text) . "%') order by " .
-				"dwell desc, parcelname" .
-				" limit ".(0+$query_start).",100");
-	}
-	else
-	{
-		$result = mysql_query("select * from parcels where " .
-				"parcelname like '%" .
-				mysql_escape_string($text) . "%' or description like '" .
-				mysql_escape_string($text) . "%' order by " .
-				"dwell desc, parcelname" .
-				" limit ".(0+$query_start).",100");
-	}
+ if ($category != -1)
+ {
+ $result = mysql_query("select * from parcels where " .
+           "(searchcategory = -1 or searchcategory = '" .
+           mysql_escape_string($category) ."') and (parcelname like '%" .
+           mysql_escape_string($text) . "%' or description like '" .
+           mysql_escape_string($text) . "%') order by " .
+           "dwell desc, parcelname" .
+           " limit ".(0+$query_start).",100");
+ }
+ else
+ {
+ $result = mysql_query("select * from parcels where " .
+           "parcelname like '%" .
+           mysql_escape_string($text) . "%' or description like '" .
+           mysql_escape_string($text) . "%' order by " .
+           "dwell desc, parcelname" .
+           " limit ".(0+$query_start).",100");
+ }
 
-	$data = array();
-	while (($row = mysql_fetch_assoc($result)))
-	{
-		$data[] = array(
-				"parcel_id" => $row["infouuid"],
-				"name" => $row["parcelname"],
-				"for_sale" => "False",
-				"auction" => "False",
-				"dwell" => $row["dwell"]);
-	}
-	$response_xml = xmlrpc_encode(array(
-		'success'	  => True,
-		'errorMessage' => "",
-		'data' => $data
-	));
+ $data = array();
 
-	print $response_xml;
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "parcel_id" => $row["infouuid"],
+           "name" => $row["parcelname"],
+           "for_sale" => "False",
+           "auction" => "False",
+           "dwell" => $row["dwell"]
+           );
+ }
+
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+ print $response_xml;
 }
 
 xmlrpc_server_register_method($xmlrpc_server, "dir_popular_query",
-		"dir_popular_query");
+                "dir_popular_query");
 
 function dir_popular_query($method_name, $params, $app_data)
 {
-	$req	= $params[0];
+ $req            = $params[0];
 
-	$flags	= $req['flags'];
+ $flags          = $req['flags'];
 
-	$terms	= array();
+ $terms = array();
 
-	if ($flags & 0x1000)
-		$terms[] = "has_picture = 1";
+ if ($flags & 0x1000)
+     $terms[] = "has_picture = 1";
 
-	if ($flags & 0x0800)
-		$terms[] = "mature = 0";
+ if ($flags & 0x0800)
+     $terms[] = "mature = 0";
 
-	$where = "";
-	if (count($terms) > 0)
-		$where = " where " . join(" and ", $terms);
+ $where = "";
 
-	$result = mysql_query("select * from popularplaces" . $where);
+ if (count($terms) > 0)
+     $where = " where " . join(" and ", $terms);
 
-	$data = array();
-	while (($row = mysql_fetch_assoc($result)))
-	{
-		$data[] = array(
-				"parcel_id" => $row["infoUUID"],
-				"name" => $row["name"],
-				"dwell" => $row["dwell"]);
-	}
+ $result = mysql_query("select * from popularplaces" . $where);
 
-	$response_xml = xmlrpc_encode(array(
-			'success'	  => True,
-			'errorMessage' => "",
-			'data' => $data));
+ $data = array();
 
-	print $response_xml;
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "parcel_id" => $row["infoUUID"],
+           "name" => $row["name"],
+           "dwell" => $row["dwell"]
+           );
+ }
+
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+ print $response_xml;
 }
 
 xmlrpc_server_register_method($xmlrpc_server, "dir_land_query",
-		"dir_land_query");
+                "dir_land_query");
 
 function dir_land_query($method_name, $params, $app_data)
 {
-	$req			= $params[0];
+ $req            = $params[0];
 
-	$flags			= $req['flags'];
-	$type			= $req['type'];
-	$price			= $req['price'];
-	$area			= $req['area'];
-	$query_start	= $req['query_start'];
+ $flags          = $req['flags'];
+ $type           = $req['type'];
+ $price          = $req['price'];
+ $area           = $req['area'];
+ $query_start    = $req['query_start'];
 
-	$terms = array();
-	$order = "lsq";
-	if ($flags & 0x80000)
-		$order = "parcelname";
-	if ($flags & 0x10000)
-		$order = "saleprice";
-	if ($flags & 0x40000)
-		$order = "area";
-	if (!($flags & 0x8000))
-		$order .= " desc";
+ $terms = array();
 
-	if ($flags & 0x100000)
-		$terms[] = "saleprice <= '" . mysql_escape_string($price) . "'";
+ $order = "lsq";
 
-	if ($flags & 0x200000)
-		$terms[] = "area >= '" . mysql_escape_string($area) . "'";
+ if ($flags & 0x80000)
+     $order = "parcelname";
 
-	if (($type & 26) == 2) // Auction
-	{
-		$response_xml = xmlrpc_encode(array(
-				'success' => False,
-				'errorMessage' => "No auctions listed"));
+ if ($flags & 0x10000)
+     $order = "saleprice";
 
-		print $response_xml;
+ if ($flags & 0x40000)
+     $order = "area";
 
-		return;
-	}
+ if (!($flags & 0x8000))
+       $order .= " desc";
 
-	if (($type & 24) == 8)
-		$terms[] = "parentestate = 1";
-	if (($type & 24) == 16)
-		$terms[] = "parentestate <> 1";
+ if ($flags & 0x100000)
+     $terms[] = "saleprice <= '" . mysql_escape_string($price) . "'";
 
-	if ($flags & 0x800)
-		$terms[] = "mature = 'false'";
-	if ($flags & 0x4000)
-		$terms[] = "mature = 'true'";
+ if ($flags & 0x200000)
+     $terms[] = "area >= '" . mysql_escape_string($area) . "'";
 
-	$where = "";
-	if (count($terms) > 0)
-		$where = " where " . join(" and ", $terms);
+ if (($type & 26) == 2) // Auction
+ {
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => False,
+                 'errorMessage' => "No auctions listed"
+                 ));
 
-	$sql = "select *, saleprice/area as lsq from parcelsales" . $where .
-				" order by " . $order . " limit " .
-				mysql_escape_string($query_start) . ",101";
+ print $response_xml;
 
-	$result = mysql_query($sql);
+ return;
+ }
 
-	$data = array();
-	while (($row = mysql_fetch_assoc($result)))
-	{
-		$data[] = array(
-				"parcel_id" => $row["infoUUID"],
-				"name" => $row["parcelname"],
-				"auction" => "false",
-				"for_sale" => "true",
-				"sale_price" => $row["saleprice"],
-				"area" => $row["area"]);
-	}
+ if (($type & 24) == 8)
+      $terms[] = "parentestate = 1";
 
-	$response_xml = xmlrpc_encode(array(
-			'success'	  => True,
-			'errorMessage' => "",
-			'data' => $data));
+ if (($type & 24) == 16)
+      $terms[] = "parentestate <> 1";
 
-	print $response_xml;
+ if ($flags & 0x800)
+     $terms[] = "mature = 'false'";
+
+ if ($flags & 0x4000)
+     $terms[] = "mature = 'true'";
+
+ $where = "";
+
+ if (count($terms) > 0)
+     $where = " where " . join(" and ", $terms);
+
+ $sql = "select *, saleprice/area as lsq from parcelsales" . $where .
+         " order by " . $order . " limit " .
+         mysql_escape_string($query_start) . ",101";
+
+ $result = mysql_query($sql);
+
+ $data = array();
+
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "parcel_id" => $row["infoUUID"],
+           "name" => $row["parcelname"],
+           "auction" => "false",
+           "for_sale" => "true",
+           "sale_price" => $row["saleprice"],
+           "area" => $row["area"]
+           );
+ }
+
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+ print $response_xml;
 }
 
 xmlrpc_server_register_method($xmlrpc_server, "dir_events_query",
-		"dir_events_query");
+                "dir_events_query");
 
 function dir_events_query($method_name, $params, $app_data)
 {
-	$req			= $params[0];
+ $req            = $params[0];
 
-	$text			= $req['text'];
-	$flags			= $req['flags'];
-	$query_start	= $req['query_start'];
+ $text           = $req['text'];
+ $flags          = $req['flags'];
+ $query_start    = $req['query_start'];
 
-	if ($text == "%%%")
-	{
-		$response_xml = xmlrpc_encode(array(
-				'success'	  => False,
-				'errorMessage' => "Invalid search terms"
-		));
+ if ($text == "%%%")
+ {
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => False,
+                 'errorMessage' => "Invalid search terms"
+                 ));
 
-		print $response_xml;
+ print $response_xml;
 
-		return;
-	}
+ return;
+ }
 
-	$terms = array();
+ $terms = array();
 
-	if ($flags & 0x2000) $terms[] = "mature = 'false'";
+ if ($flags & 0x2000) $terms[] = "mature = 'false'";
 
-	$where = "";
-	if (count($terms) > 0)
-	$where = " where " . join(" and ", $terms);
+ $where = "";
 
-	$sql = "select * from events". $where.
-		   " limit " . mysql_escape_string($query_start) . ",101";
+ if (count($terms) > 0)
+     $where = " where " . join(" and ", $terms);
 
+ $sql = "select * from events". $where.
+         " limit " . mysql_escape_string($query_start) . ",101";
 
-	$result = mysql_query($sql);
 
-	$data = array();
-	while (($row = mysql_fetch_assoc($result)))
-	{
-		$data[] = array(
-				"owner_id" => $row["OwnerID"],
-				"name" => $row["Name"],
-				"event_id" => $row["EventID"],
-				"date" => $row["Date"],
-				"unix_time" => $row["UnixTime"],
-				"event_flags" => $row["EventFlags"]);
-	}
+ $result = mysql_query($sql);
 
-	$response_xml = xmlrpc_encode(array(
-			'success'	  => True,
-			'errorMessage' => "",
-			'data' => $data));
+ $data = array();
 
-	print $response_xml;
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "owner_id" => $row["OwnerID"],
+           "name" => $row["Name"],
+           "event_id" => $row["EventID"],
+           "date" => $row["Date"],
+           "unix_time" => $row["UnixTime"],
+           "event_flags" => $row["EventFlags"]
+           );
+ }
+
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+ print $response_xml;
 }
 
+xmlrpc_server_register_method($xmlrpc_server, "dir_classified_query",
+                "dir_classified_query");
+
+function dir_classified_query ($method_name, $params, $app_data)
+{
+ $req            = $params[0];
+
+
+ $result = mysql_query($sql);
+
+ $data = array();
+
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "ClassifiedID" => $row["ClassifiedID"],
+           "Name" => $row["Name"],
+           "ClassifiedFlags" => $row["ClassifiedFlags"],
+           "CreationDate" => $row["CreationDate"],
+           "ExpirationDate" => $row["ExpirationDate"],
+           "PriceForListing" => $row["PriceForListing"]
+           );
+ }
+
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+ print $response_xml;
+}
+
+// Events Info Query
+
 xmlrpc_server_register_method($xmlrpc_server, "event_info_query",
-		"event_info_query");
+                "event_info_query");
 
 function event_info_query($method_name, $params, $app_data)
 {
-	$req		= $params[0];
+ $req            = $params[0];
 
-	$eventID	= $req['eventID'];
+ $eventID        = $req['eventID'];
 
-	$sql =  "select * from events where eventID = " .
-			mysql_escape_string($eventID); 
+ $sql =  "select * from events where eventID = " .  mysql_escape_string($eventID);
 
-	$result = mysql_query($sql);
+ $result = mysql_query($sql);
 
-	$data = array();
-	while (($row = mysql_fetch_assoc($result)))
-	{
-		$data[] = array(
-				"event_id" => $row["eventID"],
-				"creator" => $row["Creator"],
-				"name" => $row["Name"],
-				"category" => $row["Category"],
-				"description" => $row["Desc"],
-				"date" => $row["Date"],
-				"dateUTC" => $row["DateUTC"],
-				"duration" => $row["Duration"],
-				"covercharge" => $row["Cover"],
-				"coveramount" => $row["Amount"],
-				"simName" => $row["SimName"],
-				"globalposition" => $row["GlobalPos"],
-				"eventflags" => $row["EventFlags"]);
-	}
+ $data = array();
 
-	$response_xml = xmlrpc_encode(array(
-			'success'	  => True,
-			'errorMessage' => "",
-			'data' => $data));
+ while (($row = mysql_fetch_assoc($result)))
+ {
+ $data[] = array(
+           "event_id" => $row["eventID"],
+           "creator" => $row["Creator"],
+           "name" => $row["Name"],
+           "category" => $row["Category"],
+           "description" => $row["Desc"],
+           "date" => $row["Date"],
+           "dateUTC" => $row["DateUTC"],
+           "duration" => $row["Duration"],
+           "covercharge" => $row["Cover"],
+           "coveramount" => $row["Amount"],
+           "simName" => $row["SimName"],
+           "globalposition" => $row["GlobalPos"],
+           "eventflags" => $row["EventFlags"]
+           );
+ }
 
-	print $response_xml;
+ $response_xml = xmlrpc_encode(array(
+                 'success'      => True,
+                 'errorMessage' => "",
+                 'data' => $data
+                 ));
+
+    print $response_xml;
 }
 
 #
@@ -309,4 +369,4 @@
 $request_xml = $HTTP_RAW_POST_DATA;
 xmlrpc_server_call_method($xmlrpc_server, $request_xml, '');
 xmlrpc_server_destroy($xmlrpc_server);
-?>
+?>
\ No newline at end of file
