Index: query.php
===================================================================
--- query.php	(revision 79)
+++ query.php	(working copy)
@@ -249,7 +249,7 @@
 				"name" => $row["Name"],
 				"event_id" => $row["EventID"],
 				"date" => $row["Date"],
-				"unix_time" => $row["UnixTime"],
+				"unix_time" => $row["DateUTC"],
 				"event_flags" => $row["EventFlags"]);
 	}
 
@@ -267,7 +267,26 @@
 function dir_classified_query ($method_name, $params, $app_data)
 {
 	$req			= $params[0];
+	$text			= $req['text'];
+	$flags			= $req['flags'];
+	$category		= $req['category'];
+	$query_start	= $req['query_start'];
 
+	if ($text == "%%%")
+	{
+		$response_xml = xmlrpc_encode(array(
+				'success'	  => False,
+				'errorMessage' => "Invalid search terms"
+		));
+
+		print $response_xml;
+
+		return;
+	}
+
+	$sql = "select * from classifieds
+			limit " . mysql_escape_string($query_start) . ",101";
+
 	$result = mysql_query($sql);
 
 	$data = array();
