Index: query.php
===================================================================
--- query.php	(revision 80)
+++ query.php	(working copy)
@@ -249,7 +249,7 @@
 				"name" => $row["Name"],
 				"event_id" => $row["EventID"],
 				"date" => $row["Date"],
-				"unix_time" => $row["UnixTime"],
+				"unix_time" => $row["DateUTC"],
 				"event_flags" => $row["EventFlags"]);
 	}
 
@@ -333,11 +333,10 @@
 	print $response_xml;
 }
 
-#
 # Process the request
 #
 
 $request_xml = $HTTP_RAW_POST_DATA;
 xmlrpc_server_call_method($xmlrpc_server, $request_xml, '');
 xmlrpc_server_destroy($xmlrpc_server);
-?>
+?>
\ No newline at end of file
