Index: trunk/webroot/query.php
===================================================================
--- trunk/webroot/query.php	(revision 83)
+++ trunk/webroot/query.php	(working copy)
@@ -244,12 +244,14 @@
 	$data = array();
 	while (($row = mysql_fetch_assoc($result)))
 	{
+		$row["Date"] = strftime("%m/%d %I:%M %p",$row["DateUTC"]);
+		
 		$data[] = array(
 				"owner_id" => $row["OwnerID"],
 				"name" => $row["Name"],
 				"event_id" => $row["EventID"],
 				"date" => $row["Date"],
-				"unix_time" => $row["UnixTime"],
+				"unix_time" => $row["DateUTC"],
 				"event_flags" => $row["EventFlags"]);
 	}
 
@@ -274,12 +276,12 @@
 	while (($row = mysql_fetch_assoc($result)))
 	{
 		$data[] = array(
-				"ClassifiedID" => $row["ClassifiedID"],
-				"Name" => $row["Name"],
-				"ClassifiedFlags" => $row["ClassifiedFlags"],
-				"CreationDate" => $row["CreationDate"],
-				"ExpirationDate" => $row["ExpirationDate"],
-				"PriceForListing" => $row["PriceForListing"]);
+				"classifiedid" => $row["ClassifiedID"],
+				"name" => $row["Name"],
+				"classifiedFlags" => $row["ClassifiedFlags"],
+				"creationdate" => $row["CreationDate"],
+				"expirationdate" => $row["ExpirationDate"],
+				"priceforlisting" => $row["PriceForListing"]);
 	}
 
 	$response_xml = xmlrpc_encode(array(
@@ -310,7 +312,7 @@
 	while (($row = mysql_fetch_assoc($result)))
 	{
 		$data[] = array(
-				"event_id" => $row["eventID"],
+				"event_id" => $row["EventID"],
 				"creator" => $row["Creator"],
 				"name" => $row["Name"],
 				"category" => $row["Category"],
