Index: register.php
===================================================================
--- register.php	(revision 93)
+++ register.php	(working copy)
@@ -16,8 +16,9 @@
 
 $hostname = $_GET['host'];
 $port = $_GET['port'];
+$service = $_GET['service'];
 
-if ($hostname != "" && $port != "")
+if ($hostname != "" && $port != "" && $service == "online")
 {
     // Check if there is already a database row for this host
     $checkhost = mysql_query("SELECT register from hostsregister where " .
@@ -47,5 +48,12 @@
 		$runupdate = mysql_query($register);
 	}
 }
+elseif ($hostname != "" && $port != "" && $service = "offline")
+{
+        $delete = "DELETE FROM hostsregister " .
+                "WHERE host = '" . mysql_escape_string($hostname) . "' AND " .
+                "port = '" . mysql_escape_string($port) . "'";
 
-?>
+        $rundelete = mysql_query($delete);
+}
+?>
\ No newline at end of file
